tools/WinObjC.Tools/vsimporter-templates/packageable.project.json

```diff
@@ -0,0 +1,21 @@
+{
+  "dependencies": {
+    "WinObjC.Packaging": "*",
+    "WinObjC.Frameworks": "*",
+    "WinObjC.Frameworks.UWP": "*",
+    "WinObjC.Frameworks.Core": "*",
+    "WinObjC.Language": "*",
+    "WinObjC.Logging": "*"
```

Comment:
To be decided. I was having some difficulties since these packages were all being built as prereleases. Need input from Brian.

---
In reply to: [98955335](https://github.com/Microsoft/WinObjC/pull/1880#discussion_r98955335) [](ancestors = 98955335)